packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -26,7 +25,7 @@ namespace Microsoft.Generator.CSharp.ClientModel.Providers
     /// <summary>
     /// This class provides the set of serialization models, methods, and interfaces for a given model.
     /// </summary>
-    internal class MrwSerializationTypeDefinition : TypeProvider
+    public class MrwSerializationTypeDefinition : TypeProvider
```

Comment:
could you specify what we are expecting to change in this class after we make it public?

Reply:
Please check https://github.com/Azure/azure-sdk-for-net/pull/47730#discussion_r1912931489

Reply:
then it should be fine